examples/conftest/README.md

```diff
@@ -0,0 +1,110 @@
+# Conftest Example
+
+This example shows how to use Atlantis' built in [Conftest](https://www.conftest.dev/) support with Infracost to enforce cost policies.
```

Comment:
```suggestion
This example shows how to use [Atlantis' built in Conftest](https://www.runatlantis.io/docs/policy-checking.html) support with Infracost to enforce cost policies.
```

I thought this might be better as it shows how they can enable the built-in Conftest